CHANGES.rst

```diff
@@ -25,6 +25,7 @@ Improvements
   restrict them to speakers or registrants (:pr:`5670`, thanks :user:`kewisch`)
 - Make editing timeline display much more straightforward (:pr:`5674`)
 - Allow event managers to delete editables from contributions (:pr:`5778`)
+- Add `lang` attribute to `<html>` elements in base templates (:pr:`5894`)
```

Comment:
rst uses double backticks for code highlighting (single backticks are already used for rst references and similar features)